examples/webhook.ts

```diff
@@ -0,0 +1,176 @@
+import { SpvWalletClient, WebhookManager, WebhookHttpHandler, RawEvent } from '../dist/typescript-npm-package.cjs.js';
+import { exampleAdminKey } from './example-keys.js';
+import { errMessage } from './utils.js';
+// import http from 'http';
+// import express, { Response, Request } from 'express';
+import Fastify, { FastifyReply, FastifyRequest } from 'fastify';
+
+// NOTE: ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+// THIS IS AN EXAMPLE ON HOW YOU CAN SET UP A WEBHOOK NOTIFIER
+// It is made in such a way that it is possible to use any lib you want for sending and reveiving requests
```

Comment:
What do you mean by `possible to use any lib .... for SENDING` requests` ?
The `fastify`/`express` stuff is rather only for receiving requests/notifications.

Reply:
You're right..and not :D I meant here that you can use the lib you want to receive something from the /notification url but also can send back the result. I will delete sending part